site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -103,6 +104,15 @@
         </div>
       </div>
 
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">Project Classification</h3>
+        </div>
+        <div class="panel-body">
+          {{ form_row(form.classifications) }}
```

Comment:
This will not work without javascript. Don't render it. Instead construct a dropdown that you manually populate, just use the `name` from the form. 

```html
<select name="{{ form.classifications.name }} " ... >
   <option value="...">Text</option>
</select>
```

Reply:
https://symfony.com/doc/current/form/form_collections.html#allowing-new-tags-with-the-prototype